hathor/manager.py

```diff
@@ -703,6 +710,31 @@ def generate_parent_txs(self, timestamp: Optional[float]) -> 'ParentTxs':
         can_include = [i.data for i in can_include_intervals]
         return ParentTxs(max_timestamp, can_include, must_include)
 
+    def get_best_blockchain(self, n_blocks: int) -> list['BlockInfo']:
+        """ Get a list with N blocks from the best blockchain
+        in descending order.
+        """
+        if not (n_blocks > 0):
+            raise ValueError(
+                'Invalid N. N must be greater than 0.'
+            )
+
+        block = self.tx_storage.get_best_block()
+        if self._latest_best_blockchain:
+            best_block = self._latest_best_blockchain[0]
+            if block.hash_hex == best_block.hash_hex and n_blocks <= len(self._latest_best_blockchain):
```

Comment:
Why are you using `hash_hex`? I feel that `block.hash` is the way to compare.

Reply:
Sure, binary comparison is better in this case. I will change.

Reply:
Oh. I saw now. I have used the `hash_hex` because the `BlockInfo` only has `hash_hex`.